.github/workflows/end-to-end.yml

```diff
@@ -0,0 +1,9 @@
+name: End-to-end tests
+on:
+  pull_request:
```

Comment:
In the future this will get updated to run on a label on a PR, cron job, manual run etc.